src/constants/packages.ts

```diff
@@ -1,64 +1,71 @@
 import { ContributeType, Goal } from '../types/contribute';
 
+/**
+ * This file contains the different contribution packages.
+ * Note that the name and text should contain the keys for
+ * translations found in the
+ * public/static/locale/current_language/contribute.json file
+ */
+
 export const speakGoals: Goal[] = [
     {
         contributeType: ContributeType.SPEAK,
-        name: 'Lítið',
-        text: '10 setningar',
+        name: 'package-select.speak.small',
+        text: 'package-select.speak.text',
         count: 10,
     },
     {
         contributeType: ContributeType.SPEAK,
-        name: 'Miðlungs',
-        text: '20 setningar',
+        name: 'package-select.speak.medium',
+        text: 'package-select.speak.text',
         count: 20,
     },
     {
         contributeType: ContributeType.SPEAK,
-        name: 'Mikið',
-        text: '50 setningar',
+        name: 'package-select.speak.large',
+        text: 'package-select.speak.text',
```

Comment:
Does this distinct between 10, 20 or 50 sentences? Because it is always 'package-select.repeat.text' for text while name does specify small, medium or large.

Reply:
The way it works is that the **package-select.speak.text** string finds this text in the translations file:
![image](https://user-images.githubusercontent.com/72071258/126488275-9c42d7fa-7914-4644-bc67-eab557d6d3c1.png)
Then to get the numbers in we take that from the packages count and enter it there
![image](https://user-images.githubusercontent.com/72071258/126488740-9e4ed6cf-1a77-492c-8676-e665011024b8.png)


So basically the .text has a space for inputs that we use to set the count.

Reply:
Super cool!